andela/suyabay
# Fix #121655437] Extend player to all episodes

#### What does this PR do?

This PR ensures that the audio player is included for all podcasts.
#### Description of Task to be completed?

Extend the audio payer for podcasts to every page, included the main page, the page for all episodes and the view for all the episodes belonging to a channel.
#### How should this be manually tested?

> On the home page, notice that each podcass has an audio player included.
> On the view for all episodes, or the view for the episodes in one channel, notice that the audio player is also included.
> Any changes to the view for a podcasts will only be done one on blade partial file.
#### Any background context you want to provide?

The reason for this PR is to enable the user listen to any podcasts from the home page.
#### What are the relevant pivotal tracker stories?

The related PT story can be viewed [here](https://www.pivotaltracker.com/story/show/121655437);
#### Screenshots (if appropriate)

![podplayers](https://cloud.githubusercontent.com/assets/17028608/16114744/c636d70a-33ba-11e6-90bf-2b30f8438d30.png)
#### Questions:

The home view separated the first episode from all other episodes. I think we could apply this to the episode that has mostly been liked.

[@unicodeveloper](https://github.com/unicodeveloper)
